api/lib/application/authentication/oidc/oidc-controller.js

```diff
@@ -19,6 +19,30 @@ module.exports = {
     return h.response({ redirectLogoutUrl }).code(200);
   },
 
+  async findUserForReconciliation(request, h) {
+    const { email, password, identityProvider, authenticationKey } = request.deserializedPayload;
+    const oidcAuthenticationService = authenticationRegistry.lookupAuthenticationService(identityProvider);
+
+    const result = await usecases.findUserForOidcReconciliation({
+      email,
+      password,
+      identityProvider,
+      authenticationKey,
+      oidcAuthenticationService,
+    });
+
+    if (result.isAuthenticationComplete) {
+      return h
+        .response({
+          access_token: result.pixAccessToken,
+          logout_url_uuid: result.logoutUrlUUID,
+        })
+        .code(200);
+    } else {
+      return h.response().code(204);
+    }
```

Comment:
```suggestion
    
    return h.response().code(204);
```

le else n'est pas necessaire puisque si on tombe dans le if, il y a un return